### PR TITLE
add anomality for main search engine

### PR DESCRIPTION
**Pull request by xluo-ds**
original look at over all engines, now add anomaly for Google/Bing/DDG/Yandex

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is
  referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to
  `.circleci/config.yml`) will cause environment variables (particularly
  credentials) to be exposed in test logs
- [ ] Ensure the container image will be using permissions granted to
  [telemetry-airflow](https://github.com/mozilla/telemetry-airflow/)
  responsibly.

### CODE REVIEW

**Comment by xluo-ds** [comment]
@akkomar  somehow I can't add Reviewers in the right side bar so tag you here. Thanks!

File: jobs/search-alert/search_alert/main.py [hunk 1]
```diff
@@ -55,17 +56,20 @@ def main(project_id, submission_date, dry_run):
             submission_date = @submission_date
         GROUP BY
             1,
-            2 ),
+            2,
+	    3),
```
Comment (akkomar) [inline jobs/search-alert/search_alert/main.py:60]:
```suggestion
	        3 ),
```

File: jobs/search-alert/search_alert/main.py [hunk 2]
```diff
@@ -55,17 +56,20 @@ def main(project_id, submission_date, dry_run):
...
         long_data AS (
         SELECT
             submission_date,
             country,
+	    engine,
```
Comment (akkomar) [inline jobs/search-alert/search_alert/main.py:65]:
```suggestion
	        engine,
```

File: jobs/search-alert/search_alert/main.py [hunk 3]
```diff
@@ -55,17 +56,20 @@ def main(project_id, submission_date, dry_run):
...
             metric,
             value
         FROM (
             SELECT
             submission_date,
             country,
+	    engine,
```
Comment (akkomar) [inline jobs/search-alert/search_alert/main.py:72]:
```suggestion
	        engine,
```

File: jobs/search-alert/search_alert/main.py [hunk 4]
```diff
@@ -98,23 +105,52 @@ def main(project_id, submission_date, dry_run):
...
+	    FROM long_data
+	    WHERE engine in ("Google", "Bing", "DuckDuckGo", "DDG", "Yandex")
+	    )),	
         extended_new_data AS (
         SELECT
             submission_date,
             country,
             new_data_index,
             metric,
+	    engine,
```
Comment (akkomar) [inline jobs/search-alert/search_alert/main.py:145]:
```suggestion
	        engine,
```

File: jobs/search-alert/search_alert/main.py [hunk 5]
```diff
@@ -127,9 +163,10 @@ def main(project_id, submission_date, dry_run):
             country,
             TRUE AS new_data_index,
             metric,
-            value
+            engine, 
+	    value
```
Comment (akkomar) [inline jobs/search-alert/search_alert/main.py:167]:
```suggestion
	        value
```

**Review by akkomar — CHANGES_REQUESTED** [review]
My only request is to make formatting consistent here (see suggested changes).

This looks good to me. I'm a bit worried that we're appending twice to the `desktop_search_alert_records` here and this job is not idempotent - I should spend some time to improve that.

**Pull request closed without merging**